modules/manage/pages/security/authorization/acl.adoc

```diff
@@ -178,6 +194,64 @@ ADMIN
 
     AlterConfigs      ALTER_CONFIGS on CLUSTER for cluster (broker altering)
                       ALTER_CONFIGS on TOPIC for topics (topic altering)
+
+    AlterPartitionAssignments   ALTER on CLUSTER for kafka-cluster
```

Comment:
@Feediver1 I guess the partially capitalized are Kafka API names.  The full caps is our 'operation'